hugr-core/src/hugr/serialize.rs

```diff
@@ -166,6 +166,21 @@ impl Hugr {
     }
 }
 
+/// Deerialize the HUGR using a serde decoder.
+///
+/// This API is unstable API and will be removed in the future.
+#[deprecated(
+    since = "0.20.0",
+    note = "This API is unstable and will be removed in the future.
+            Use `Hugr::load` or the `AsStringEnvelope` adaptor instead."
+)]
+pub fn serde_deserialize_hugr<'de, D>(deserializer: D) -> Result<Hugr, D::Error>
```

Comment:
IIUC this is a new deprecated method to replace the old `Hugr::serde_deserialize` - my concern is that the discoverability of this relative to using the old method isn't high enough to justify it